### ISSUE TITLE
[Issue] - bad argument #1 to 'vector3' (invalid vector type)

### ISSUE THREAD

**Issue by NocityRP** [hoangducdt/MojiaGarages#51]
Getting this error when starting my city:

![image](https://user-images.githubusercontent.com/93712286/149399046-ba2ab5c0-0ed3-48d2-a05b-2c6f747ea0ec.png)

Then getting this in the city, when inside a PolyZone:

![image](https://user-images.githubusercontent.com/93712286/149399119-7af08b05-cc21-44ff-a224-3934563154aa.png)

**Comment by meistroburger** [comment]
ReadMe is not up to date

**Comment by NocityRP** [comment]
> ReadMe is not up to date

I've done everything as asked, and still getting it, show I go back through the readme? and make sure everything is added?

**Comment by BlueSky-fur** [comment]
same issue, with the latest ReadMe. also the mysql string for the qb-phone edit seems to be wrong, and the sql-file seems to have issues too.

**Comment by hoangducdt** [comment]
> same issue, with the latest ReadMe. also the mysql string for the qb-phone edit seems to be wrong, and the sql-file seems to have issues too.

update to latest qb-phone

**Comment by BlueSky-fur** [comment]
all resources are up-to-date, the error still persists.
https://gyazo.com/646b2427c53816b809313f3b3f15d7fa

**Comment by GoldenGramsTTV** [comment]
Having the same issue. I'll be spending the next few hours trying to come to a solution. Everything seems to be fine outside of vehicles, as soon as you are in a vehicle it starts spamming f8 menu with that error

**Comment by hoangducdt** [comment]
strange that i tried on original qb-core and no problem.

**Comment by GoldenGramsTTV** [comment]
Yeah I'm stumped. I been up almost 2 days now so I'm gunna get some sleep and maybe try it again when I wake up.

Maybe I'll have to do a fresh install of the garages again, idk

**Comment by BlueSky-fur** [comment]
> strange that i tried on original qb-core and no problem.

ive ran a completly clean QB-Core install yesterday and it spit out the errors mentioned in my last message

**Comment by dragon703** [comment]
So I was running into the same issue, but fixed it when I realized that this resource doesn't work well with previous databases which has saved cars, house locations and garages saved to it. So upon resetting the database completely... the issue was solved.
